joomla/joomla-cms
# [4.0] Fixed delete installation folder function

Fixes https://github.com/joomla/joomla-cms/issues/18765 and https://github.com/joomla/joomla-cms/issues/25610

## Summary of Changes
Fixed the remove installation folder function and change the checkbox for removing the folder to an button.
![pr_remove_install_folder](https://user-images.githubusercontent.com/20770106/64780852-bf53e080-d561-11e9-8352-cb8f8be22c08.png)

### Testing Instructions
1. Start the installation of Joomla 4 
2. finish the installation
3. On the "Congratulations"-Page, you can find a Box with the title "We detected development mode" in this box you should see a checkbox where you can select that the installation folder should be deleted after the installation
4. Now check that button and click the "Complete & Open Site / Admin"
5. If you now check your filesystem, the installation folder should still be there.

6. Now remove in your Joomla directory the configuration.php file
7. Now apply my patch
8. Redo the test instructions 
9. Now you should see a button instead of a checkbox.
10. Now click on the button
11. An "alert"-window should now open where you have to confirm that you wanna remove the installation folder
12. Confirm that
13. Now check your filesystem again
14. The installation folder should now be removed.



### Expected result
Installation Folder should be remove if checkbox is "checked"


### Actual result
Installation Folder is not removed.


### Documentation Changes Required
Maybe not sure :)

@roland-d @wilsonge
